lib/css/CssGenerator.js

```diff
@@ -32,6 +33,18 @@ class CssGenerator extends Generator {
 	generate(module, generateContext) {
 		const originalSource = /** @type {Source} */ (module.originalSource());
 		const source = new ReplaceSource(originalSource);
+
+		const exportsInfo = generateContext.moduleGraph.getExportsInfo(module);
+
+		for (const item of exportsInfo.exports) {
+			const exportInfo = exportsInfo.getReadOnlyExportInfo(item.name);
+			const used = exportInfo.getUsed(generateContext.runtime);
+
+			if (used === UsageState.Unused) {
+				// source.replace(exportInfo.range[0], exportInfo.range[1] - 1, "");
```

Comment:
since we dont have the range for the CSS block, we cant replace it :(
Need to modify CSS Parser to hold this information